<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgSettings</class>
 <widget class="QDialog" name="ProgSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>318</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="USER_BTN_GRP">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="SCAN_GRP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Scan Options</string>
   </property>
   <widget class="QLineEdit" name="START_POS_EDIT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="START_LBL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Start pos.</string>
    </property>
   </widget>
   <widget class="QLabel" name="END_LBL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>56</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>End pos.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="END_POS_EDIT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FRAME_LBL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>77</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame count</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FRAME_COUNT_EDIT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="STEP_LBL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>63</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Step size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="STEP_SIZE_EDIT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>155</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="UNIT_NS">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>40</y>
      <width>14</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>ns</string>
    </property>
   </widget>
   <widget class="QLabel" name="UNIT_NS_2">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>80</y>
      <width>14</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>ns</string>
    </property>
   </widget>
   <widget class="QLabel" name="UNIT_NS_3">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>160</y>
      <width>14</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>ns</string>
    </property>
   </widget>
   <widget class="QLabel" name="UNIT_KHZ">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>200</y>
      <width>21</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>kHz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="REP_FREQ_EDIT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>195</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FREQ_LBL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>70</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Rep. freq.</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ENHANCE_GROUP">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Img Enhance Options</string>
   </property>
   <widget class="QLineEdit" name="KERNEL_EDIT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="KERNEL_SIZE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>77</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Kernel size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LOW_IN_EDIT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LOW_OUT_EDIT">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>115</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="HIGH_IN_EDIT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>155</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="HIGH_OUT_EDIT">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>155</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="GAMMA_EDIT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LOW_IN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>low in</string>
    </property>
   </widget>
   <widget class="QLabel" name="HIGH_IN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>49</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>high in</string>
    </property>
   </widget>
   <widget class="QLabel" name="LOW_OUT">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>49</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>low out</string>
    </property>
   </widget>
   <widget class="QLabel" name="HIGH_OUT">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>56</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>high out</string>
    </property>
   </widget>
   <widget class="QLabel" name="GAMMA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>35</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LOG_EDIT">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>75</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LOG">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>21</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="TCU_GRP">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>TCU Options</string>
   </property>
   <widget class="QCheckBox" name="AUTO_REP_FREQ_CHK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto rep. freq.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SIMPLIFY_STEP_CHK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>116</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Simplify Step</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>USER_BTN_GRP</sender>
   <signal>accepted()</signal>
   <receiver>ProgSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>USER_BTN_GRP</sender>
   <signal>rejected()</signal>
   <receiver>ProgSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
